android/sdl_android/src/main/res/values/strings.xml

```diff
@@ -4,4 +4,7 @@
     <string name="lockscreen_image_description">SDL Icon</string>
     <string name="lockscreen_device_image_description">Device Icon</string>
     <string name="default_lockscreen_warning_message">Swipe down to dismiss, acknowledging that you are not the driver.</string>
+    <string name="spp_out_of_resource">Too many apps are using Bluetooth</string>
+    <string name="notification_title">SmartDeviceLink</string>
+    <string name="sdl_error_notification_channel_name">SDL Error</string>
```

Comment:
Could you please add translations to these strings in all the languages that are specified in the proposal?

Reply:
Hi @bilal-alsharifi,
The commit b626fa6 added localized strings for all 29 languages defined in MOBILE_API.
Please review. Thanks.

Reply:
Thanks, @shiniwat for adding the languages. Can we also add strings for different dialects for the same language? like
English: en_US, en_AU, en_GB, en_IN, en_SA
Spanish: es_ES, es_MX
French: fr_CA, fr_FR
Portuguese: pt_BR, pt_PT

As these variations also exist in the `MOBILE_API `

Reply:
Well, actually adding non-locale specific strings (i.e. language only) for most languages is intentional. Most languages do not differentiate short messages per locale. For instance, the message "Too many apps are using Bluetooth" should be the same for English in US, England, Australia, India, etc. French message should be the same for Canada and France, too. So I basically did not add the string per locale if the strings are the same. Obvious exception is Chinese -- Simplified Chinese and Traditional Chinese are obviously different, so I added both.

Reply:
That actually makes all since. My only concern is that it is mentioned in the accepted proposal that there should be strings for all variations including different dialects for the same language. 
Could you please make a quick proposal revision just to clarify the ["Localization of SPP error UX"](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0230-spp-resource-management-for-android.md#localization-of-spp-error-ux) section as you did in the last comment to make sure that the SDLC agrees? For more info on revising a proposal please check [this page](https://github.com/smartdevicelink/sdl_evolution/blob/master/process.md#revising-previously-accepted-proposals).

Reply:
As I re-read the "[Localization of SPP error UX](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0230-spp-resource-management-for-android.md#localization-of-spp-error-ux)" section, the section refers to a list of language enums from[ MOBILE_API](https://github.com/smartdevicelink/rpc_spec/blob/master/MOBILE_API.xml), which needs to be supported.
However, the section does not mention about implementation details. As far as we can support all language enums that [MOBILE_API](https://github.com/smartdevicelink/rpc_spec/blob/master/MOBILE_API.xml) refers to, we should be OK.
Adding locale-specific language (e.g. Simplified Chinese) or language only (e.g. English for all English-speaking locales) resource depends on how Android resource management works. Therefore, it's my understanding that current commit still follows the latest revision of the proposal and no need to revise.

Reply:
Okay, thanks for clarifying.